code/extensions/Lumberjack.php

```diff
@@ -147,4 +147,20 @@ protected function shouldFilter() {
 			&& in_array($controller->getAction(), array("treeview", "listview", "getsubtree"));
 	}
 
+	/**
+	 * Checks config for a specified child_classname on the class extending Lumberjack.
+	 * Uses SiteTree if none found in config.
+	 *
+	 * @return string
+	 */
+	protected function getChildClassName() {

```

Comment:
is there a way to be clever about this? I'm thinking, if the class has an `allowed_children` with only one class, then infer that?